shell/browser/ui/views/menu_delegate.cc

```diff
@@ -105,11 +105,6 @@ void MenuDelegate::WillHideMenu(views::MenuItemView* menu) {
 void MenuDelegate::OnMenuClosed(views::MenuItemView* menu) {
   for (Observer& obs : observers_)
     obs.OnMenuClosed();
-
-  // Only switch to new menu when current menu is closed.
-  if (button_to_open_)
-    button_to_open_->Activate(nullptr);
```

Comment:
why is this activation call no longer needed ?

Reply:
The application crashed in this part after the new logic was implemented. Upon reviewing the method calls, it appears that the nullptr for the event does not have any effect, as I understand it.

After testing the menu behavior, I found that removing this code does not affect anything.

I might be mistaken, so if you could explain the possible effects of this code, I would appreciate it. Perhaps I should consider reverting it back then.